### PR TITLE
[Blog Post - WG App Delivery] Platform Engineering in 2024 - Industry Trends and Emerging Focus. An holistic proposal for Internal Developer Platforms - Platform Engineering ++changed internal platform to internale developer platform in the para…

### PR DESCRIPTION
**Pull request by giulioroggero**
Hi all! after some weeks and the super useful feedback from a lot of people (thanks!) here is the PR for the blog post discussed in the issue https://github.com/cncf/tag-app-delivery/issues/586

I hope that I added all file correctly :-)

Thanks for the support

Note: I deleted the previous PR #695 because I haven't signed the push

### CODE REVIEW

**Review by abangser — COMMENTED** [review]
This is great! I haven't reviewed the content since Google Docs but I know you have a lot of reviewers so happy for them to comment here. I am just messaging on some of the tech for the blog here. Very excited to see this posted and thank you so much for your hard work on it!

File: website/content/en/blog/2024-06-20-platform-engineering-pp.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
+---
+title:  Platform Engineering in 2024, Industry Trends and Emerging Focus (An holistic proposal for Internal Developer Platforms named Platform Engineering ++)
+slug:   platform-engineering-in-2024-industry-trends-and-emerging-focus-an-holistic-proposal-for-internal-developer-platforms-platform-engineering-++
+date:   2024-06-20 12:00:00 +0000
```
[the remark this answers is not in the document]
Comment (giulioroggero) [reply website/content/en/blog/2024-06-20-platform-engineering-pp.md]: thanks, updated

File: website/content/en/blog/2024-06-20-platform-engineering-pp.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
+---
+title:  Platform Engineering in 2024, Industry Trends and Emerging Focus (An holistic proposal for Internal Developer Platforms named Platform Engineering ++)
+slug:   platform-engineering-in-2024-industry-trends-and-emerging-focus-an-holistic-proposal-for-internal-developer-platforms-platform-engineering-++
```
[the remark this answers is not in the document]
Comment (giulioroggero) [reply website/content/en/blog/2024-06-20-platform-engineering-pp.md]: thanks, updated

**Comment by dstengle** [comment]
@giulioroggero I've read through the post and I like the overall direction. The library metaphor doesn't quite make it for me, but that may just be my opinion.

I do like the way you are pushing to bring more to the table with more than just infrastructure provisioning and app deployment.

Maybe you could have a first line be something along the lines of this to complement the title: "Should internal platforms be limited to self-service infrastructure provisioning and app deployment?"

**Comment by giulioroggero** [comment]
@dstengle good suggestion

**Review by roberthstrand — APPROVED** [review]

**Pull request merged**